### ISSUE TITLE
urlAnchor.href is assigned to itself

### ISSUE THREAD

**Issue by mledoze** [madrobby/zepto#1085]
In [src/ajax.js#L199](https://github.com/madrobby/zepto/blob/2bd5d7a09ba35a44e0fd2cf43c55bcab55fb3bd7/src/ajax.js#L199)

why is `urlAnchor.href` assigned to itself?

Thank you.

**Comment by mislav** [comment]
Good question. Just by reading the code, it seems pointless. [What I do in these situations](http://mislav.uniqpath.com/2014/02/hidden-documentation/) is look at git history for the line in question.

1. From the page that you linked, you can scroll to top and click on "Blame" view. All lines of code should now be annotated with commits they were last added/edited in.
2. You can see that the line you're wondering about was added in commit 0c78b8975fd430d1a0bb66fb4eb51821f226924a: "IE only cleans up URL for .href"
3. Following the link to that commit, you might hope that the commit message explains the change. This one does not. /cc @mastahyeti 
4. However, GitHub will link you to the pull request in which this commit has been added. Pull requests often have additional context and discussion.

  <img width=301 src="https://cloud.githubusercontent.com/assets/887/6716631/035e4672-cd64-11e4-9f3f-d944f20e658a.png">

5. In #1049, you can see the comment "IE is great!" with the accompanying screenshot. The screenshot demonstrates a problem specific to IE that's only being solved by assigning `.href` to the value of self.

Basically, it's a workaround for an obscure IE-specific URL parsing problem.

**Comment by mledoze** [comment]
I know about the blame feature in VCS systems but did not know that I was possible to use it on Github. Thank you for your thorough explanation. I'll send a pull request to add a comment on this line to avoid future question about it.
